internal/kibana/synthetics/schema.go

```diff
@@ -55,13 +58,13 @@ type tfHTTPMonitorFieldsV0 struct {
 }
 
 type tfTCPMonitorFieldsV0 struct {
-	Host                  types.String   `tfsdk:"host"`
-	SslVerificationMode   types.String   `tfsdk:"ssl_verification_mode"`
-	SslSupportedProtocols []types.String `tfsdk:"ssl_supported_protocols"`
-	CheckSend             types.String   `tfsdk:"check_send"`
-	CheckReceive          types.String   `tfsdk:"check_receive"`
-	ProxyURL              types.String   `tfsdk:"proxy_url"`
-	ProxyUseLocalResolver types.Bool     `tfsdk:"proxy_use_local_resolver"`
+	Host                  types.String `tfsdk:"host"`
+	SslVerificationMode   types.String `tfsdk:"ssl_verification_mode"`
+	SslSupportedProtocols types.List   `tfsdk:"ssl_supported_protocols"`
```

Comment:
This type change for the field potentially breaks existing users of the Terrafor module. Are these resources marked such that we can change this? I'm going to presume that we can do this specifically because anyone that used these affected resources before have inconsistent states anyway and will need to update to this version.

Reply:
Both these types would be represented as a json list of strings in state either way so the change should be compatible.

@biscout42 iiuc fully specifying these fields in the TF module would avoid the error. Are we able to verify state compatibility in an acceptance test which first creates the resource with provider v0.11.7 and the reapplies with the current version ([example](https://github.com/elastic/terraform-provider-elasticstack/blob/b6434baddcfd07bec06653f380d4795e36bf99df/internal/elasticsearch/index/index/acc_test.go#L80))

Reply:
@devon-kim , this type change doesn't affect schema. There are TF acceptance tests, which show that config for ssl protocols list stays the same.